addons/web/static/src/views/relational_model.js

```diff
@@ -1061,10 +1061,7 @@ export class Record extends DataPoint {
                 }
 
                 if (["res.users", "res.partner"].includes(definition.comodel)) {
-                    widget =
-                        definition.type === "many2one"
-                            ? "many2one_avatar"
-                            : "many2many_tags_avatar";
+                    widget = this._getAvatarPropertyWidget(definition);
```

Comment:
not sure to follow you, you want to instanciate a different widget ? note that here it's for the list view, not the form view, so it should use the same widget as normal field

Reply:
No, not the widget as component, the widget as variable in the code here above :)

Instead of hooking just those 3 lignes, move the whole computation of widget into its sub method. Here you hook a part of the computation, but it is better to have a method for the complete widget computation (from line 1061 to 1074). That way people might use it / extend it  completely, instead of having just part of it that is exposed.